pkg/skaffold/build/cluster/cluster.go

```diff
@@ -42,7 +46,46 @@ func (b *Builder) Build(ctx context.Context, out io.Writer, tags tag.ImageTags,
 		defer teardownDockerConfigSecret()
 	}
 
-	return build.InParallel(ctx, out, tags, artifacts, b.buildArtifactWithKaniko)
+	var kanikoArtifacts []*latest.Artifact
+	var otherArtifacts []*latest.Artifact
+
+	for _, a := range artifacts {
```

Comment:
Can we refactor this out to another function say
```
kaninoArtifacts, otherArtifacts := segregateArtifacts(artifacts)
```

That ways you add tests for this `segregateArtifacts`

Reply:
Thanks, done.